subjects/fromto/README.md

```diff
@@ -0,0 +1,48 @@
+## fromto
+
+### Instructions
+
+Write a function that takes two `integers` and returns a `string` representing the range of numbers from the first to the second.
```

Comment:
Suggestion: "(..) `string` showing the range of numbers from the first to the second.

Reply:
It is changed and fixed.